ci-tools/release/tools/rtl_hash.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# Licensed under the Apache-2.0 license
+
+# This tool is used to generate a hash over the Caliptra RTL files
+# This can be used to verify the integrity of the RTL after it has been integrated
+# The hash for the release is included in the release_notes.txt
+# 
+# Usage: rtl_hash.sh <path_to_rtl_src_dir> <rtl_file_list>
+#   path_to_rtl_src_dir     Path to the Caliptra RTL files. This is the src/ directory within 
+#                           the caliptra RTL repo. This may differ once integrated
+#   rtl_file_list           This list of all the files that should be included in the hash
+#                           is generated and packaged with the release (rtl_hash_file_list.txt)
+#
+# Certain files are expected to be modified during integration. These are mentioned in
+# the CaliptraIntegrationSpecification.md in the RTL repo and are excluded from the rtl hash
+# file list used to generate the RTL hash
+#
+# If any files in the list are not found, an error will be output and mention all missing files
+# The hash will not be computed since it will not match the release hash with any files missing
+
+# Exit and report failure if anything fails
+set -euo pipefail
+
+# Check arg count
+if [ $# -ne 2 ]
+  then
+    echo "Usage: $(basename $0) <path_to_rtl_src_dir> <rtl_file_list>"
+	exit -1
+fi
+
+# Get args
+rtl_path=$1
+expected_file_list=$2
+
+# Read expected file list
+expected_file_list=$(cat $expected_file_list)
+# Prepend rtl path
+expected_file_list=$(echo "$expected_file_list" | sed "s@^@"$rtl_path"/@")
```

Comment:
Why not replace the above 7 lines with:

```
expected_file_list=$(cat "$1" | sed "s@^@"$rtl_path"/@")
```

Reply:
Good point I can make that more concise.